### PR TITLE
Add usage for node

### PR DESCRIPTION
**Pull request by ikarienator**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,17 +24,25 @@ npm install shift-parser
...
 ```
 
-```js
+```es6
 import {parseScript, parseModule} from "shift-parser";
 let scriptAST = parseScript("/* ECMAScript Script text */");
 let moduleAST = parseModule("/* ECMAScript Module text */");
 ```
 
+Loading it from node.js:
```
Comment (michaelficarra) [inline README.md]: "Or in node.js:"

File: README.md [hunk 2]
```diff
@@ -24,17 +24,25 @@ npm install shift-parser
...
+
+```js
+var parseScript = require('shift-parser').parserScript;
```
Comment (michaelficarra) [inline README.md]: Typo: parserScript

File: README.md [hunk 3]
```diff
@@ -24,17 +24,25 @@ npm install shift-parser
...
+
+console.log(parseScript('/* ECMAScript Script text */'));
```
Comment (michaelficarra) [inline README.md]: Don't use console, just create a variable like in the other examples.

**Comment by ikarienator** [comment]
Amended, please review.

**Pull request merged**